src/api/app/controllers/webui/package_controller.rb

```diff
@@ -698,21 +698,26 @@ def save_file
       # We have to check if it's an AJAX request or not
       if request.xhr?
         flash.now[:success] = message
-        render layout: false, partial: 'layouts/webui/flash', object: flash
+        status = 200
       else
         redirect_to({ action: :show, project: @project, package: @package }, success: message)
+        return
       end
-      return
     else
       message = "Error while creating '#{filename}' file: #{errors.compact.join("\n")}."
       # We have to check if it's an AJAX request or not
       if request.xhr?
         flash.now[:error] = message
-        render layout: false, status: 400, partial: 'layouts/webui/flash', object: flash
+        status = 400
       else
         redirect_back(fallback_location: root_path, error: message)
+        return
       end
     end
+
+    switch_to_webui2
+    namespace = switch_to_webui2? ? 'webui2' : 'webui'
```

Comment:
Should we maybe document this in https://github.com/openSUSE/open-build-service/wiki/How-to-work-in-Bootstrap-%28new-UI%29? :thinking:

Reply:
I added it in https://github.com/openSUSE/open-build-service/wiki/How-to-work-in-Bootstrap-(new-UI)#implementation-details